release.yaml

```diff
@@ -9,12 +9,16 @@ rancher-aks-operator-crd:
 rancher-csp-adapter:
   - 1.0.1
 rancher-backup:
-- 2.1.4-rc1
+  - 2.1.4-rc1
 rancher-backup-crd:
-- 2.1.4-rc1
```

Comment:
these should be removed and this will be refactored in https://github.com/rancher/charts/pull/2095. Wait for that to merge before merging this PR